students/kosarzewski_maciej/lesson_01_basics/fractional_part.py

```diff
@@ -0,0 +1,4 @@
+# Read a float:
+a = float(input())
+# Print a value
+print (a - int(a))
```

Comment:
PEP8 warning

Please check PEP8 warnings with 'Code' -> 'Inspect code'

Reply:
For my PyCharm is ok :\

Reply:
There should be no space after print statement. For some reason your pyCharm doesn't check PEP8. This time it's ok but try to follow PEP8 style guide next time.